### DISCUSSION TITLE
Recapitation - sim_ancestry(), ploidy

### DISCUSSION THREAD

**Discussion by csmcal** [tskit-dev/pyslim#165] [Q&A] [answered]
I had a couple questions on recapitation:

1) In using recapitate(), I attempted to pass an argument dictionary:
`recap_kwargs = {'gene_conversion_rate':gc_rate,'gene_conversion_tract_length':gc_tract_len}`
However, this gave an error message:
`line 465, in recapitate
    recap = msprime.simulate(
TypeError: simulate() got an unexpected keyword argument 'gene_conversion_rate'`
So, recapitate is using simulate() instead of sim_ancestry() - is this intended? Should I try to adapt my own function based on sim_ancestry()?

2) Does specifying ploidy matter in recapitation? sim_ancestry() seems to take a default of 2, but I'm not clear

**Comment by jeromekelleher** [comment, marked as answer]
I think the plan is to release an updated version of pyslim soon that takes advantage of the new msprime 1.0 APIs @csmcal. Until then, the behaviour will be exactly the same as before, even if you have msprime 1.0 installed.

@petrelharp, is this correct?

**Reply by petrelharp** [reply]
Yep- I hope to do this in the next few weeks.

**Reply by csmcal** [reply]
Thanks for the help! Apologies for not following the versions/updates accurately
